resources/views/settings/edit.blade.php

```diff
@@ -975,5 +975,11 @@
 <script>
 //color picker with addon
 $(".header-color").colorpicker();
+// toggle the disabled state of asset id prefix
+$('#auto_increment_assets').on('ifChecked', function(){
```

Comment:
Wouldn't this need to correspond to an `id="auto_increment_asset"` and  `id="auto_increment_prefix"` on the form elements?